module-4/backend/week_one.md

```diff
@@ -5,15 +5,26 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+* Easily that I am capable of learning a new technology/language on my own. 
 2. What are some tools to help debug JavaScript code?
+* debugger, using console.log(x) like puts, and pryjs
 3. What are some tools you need in order to unit test your JavaScript?
+* mocha, chai,
 4. What is the syntax for invoking a function?
+* myFunctionName()
 5. What's `this` in JavaScript?
+* similar to 'self', 'this' is used to reference the next highest scoped object in order that you might be able to change the attributes of a thing from within.
 6. What is Webpack and why is it useful?
+* Webpack is nodejs version of a server, and allows us to test our javascript client side experience.
 7. When/why do you want to use event delegation?
+* always! use event delegation to change the page dynamically after the browser loaded by listening for user actions and performing page manipulations thereafter. 
 8. What's `npm` and what do we use it for?
+* npm stands for node package manager, and is like nodejs' version of a gemfile. 
```

Comment:
it's probably more similar to bundler than the gemfile. your package.json file is more similar to the gemfile